otaclient/app/boot_control/_common.py

```diff
@@ -394,6 +656,10 @@ class PrepareMountMixin:
     standby_slot_mount_point: Path
     ref_slot_mount_point: Path
 
+    def _prepare_mount_points(self):
+        self.standby_slot_mount_point.mkdir(exist_ok=True, parents=True)
+        self.ref_slot_mount_point.mkdir(exist_ok=True, parents=True)
+
```

Comment:
It seems this function is not referenced.

Reply:
@obi-t4  Ah yes this method is not used. 

Actually the `SlotInUseMixin` along with `PreprareMountMixin` and `OTAStatusMixin` and `VersionControlMIxin` are not used in rpi_boot, instead we use the new `OTAStatusFilesControl` and `SlotMountHelper`.

The new `OTAStatusFilesControl` and `SlotMountHelper` are scheduling to replace the old Mixins in boot_control._common also for other boot_controller implementation to reduce duplicated code and unify the status files control/slots mounting mechanism implementation. This will be a future ticket.

Reply:
> The new OTAStatusFilesControl and SlotMountHelper are scheduling to replace the old Mixins in boot_control._common also for other boot_controller implementation to reduce duplicated code and unify the status files control/slots mounting mechanism implementation.

If so, I think we don't need to modify `PreprareMountMixin`. This PR is fixing `PreprareMountMixin`.

> The deprecation of old mixins and integration of new OTAStatusFilesControl and SlotMountHelper will be in a future PR.

This strategy is OK.

Reply:
But we need a reason why those old mixin should be replaced by new OTAStatusFilesControl and SlotMountHelper.
Changing the internal design every time a new platform is supported means that the code will not be stable forever.